src/stories/CardExpirationDate.stories.tsx

```
@@ -9,18 +9,38 @@ const meta: Meta<typeof CardExpirationDate> = {
   args: {
     handleChange: () => {},
-    cardExpirationDate: ["", ""],
-    errorMessage: ["", ""],
+    handleStep: () => {},
+    step: 2,
+    cardExpirationDate,
+    errorMessage,
   },
 };
 
 export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)